pkg/kubelet/cm/cpuset/cpuset.go

```diff
@@ -46,7 +46,7 @@ func NewBuilder() Builder {
 
 // Add adds the supplied elements to the result. Calling Add after calling
 // Result has no effect.
-func (b Builder) Add(elems ...int) {
+func (b *Builder) Add(elems ...int) {
```

Comment:
This suggests a lack of tests, doesn't it?  Can we tag owners on these when we find them,

Reply:
cc @adisky -- for sig-node related tests

Reply:
yeah... I think `Add()` would actually do the right thing on a struct receiver, but `Result()` would not

Reply:
Yeah, good call about testing. There does appear to be a test [here](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cm/cpuset/cpuset_test.go#L24) but I guess it is not comprehensive enough.